src/status_im/contexts/profile/profiles/style.cljs

```diff
@@ -25,10 +25,11 @@
     :bottom   0
     :right    0}))
 
-(def profiles-header
+(defn profiles-header
+  [top]
   {:flex-direction     :row
    :padding-horizontal 20
-   :padding-top        112
+   :padding-top        (+ 56 top)
    :margin-bottom      20})
```

Comment:
Is the value `56` related to the safe-area or the design of the view?

Reply:
The design of the view hardcoded the top padding, for this case I just fixed the hardcoded value to match designs.